docs/endpoints/cookieSync.md

```diff
@@ -26,6 +27,9 @@ must supply a JSON object to define the list of bidders that may need to be sync
 If `gdpr` is  omitted, callers are still encouraged to send `gdpr_consent` if they have it.
 Depending on how the Prebid Server host company has configured their servers, they may or may not require it for cookie syncs.
 
+`limit` is optional. If present and greater than zero, it will limit the number of syncs returned to `limit`, dropping random syncs to
```

Comment:
let's not specify "dropping random syncs" here... I think that over-constrains the API a little.

For example: the only use-case (at least for now) is page performance. Clients use this to tell us how many syncs they're ok having. Given that, it may be a good idea to refresh syncs which haven't expired yet, but will expire soon, and prioritize them based on the expiry deadline.

If you specify "random", that'd be a breaking change.